RadarSDK/Radar.m

```diff
@@ -309,6 +310,26 @@ + (void)stopTrackingVerified {
     [[RadarVerificationManager sharedInstance] stopTrackingVerified];
 }
 
++ (void)startVerifyServer {
+    [[RadarLogger sharedInstance] logWithLevel:RadarLogLevelInfo type:RadarLogTypeSDKCall message:@"startVerifyServer()"];
+    [RadarUtils downloadDataFromURL:[NSURL URLWithString:@"https://s3.us-east-2.amazonaws.com/app.radar-verify.com/mac/c.der"] completionHandler:^(NSData * _Nonnull certData, NSError * _Nonnull error) {
+        [RadarUtils downloadDataFromURL:[NSURL URLWithString:@"https://s3.us-east-2.amazonaws.com/app.radar-verify.com/mac/id.p12"] completionHandler:^(NSData * _Nonnull identityData, NSError * _Nonnull error) {
+            if (!certData || !identityData) {
```

Comment:
we should be able to do these two downloads at the same time, but it's not that big of a deal since realistically it'll only be called once per session. (not worth the complication)